deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -36,9 +37,9 @@
 INVALID_CANONICAL_FEEDS_CONF = 'wazuh_invalid_canonical_feed.yaml'
 INVALID_DEBIAN_FEEDS_CONF = 'wazuh_invalid_debian_feed.yaml'
 
-REDHAT_NUM_CUSTOM_VULNERABILITIES = 6
+REDHAT_NUM_CUSTOM_VULNERABILITIES = 1
```

Comment:
It is because the custom OVAL file has one vulnerability and the vulnerabilities from the custom JSON file are not inserted in the vulnerabilities table but it fills the vulnerabilities_info table. So that is the reason. I can add more vulnerabilities to the custom OVAL file if required.